src/charts/line/utils.ts

```diff
@@ -17,20 +17,22 @@ export function getPath({
   height,
   gutter,
   shape: _shape = shape.curveBumpX,
+  yDomain,
 }: {
   data: TLineChartData;
   width: number;
   height: number;
   gutter: number;
   shape?: string;
+  yDomain: YDomain;
 }): string {
-  const timestamps = data.map(({}, i) => i);
-  const values = data.map(({ value }) => value);
+  const timestamps = data.map(({ timestamp }) => timestamp);
```

Comment:
@jxom I just realized this was a mistake, I believe it should be using the index rather than the actual time stamp. Technically I would expect both options to be possible (via a `xAxisDistributionStrategy: ‘evenly-distributed’ | ‘timestamp’`, but the previous behavior was `evenly-distributed`. All we need to do to retain previous behavior is return `i` rather than timestamp here. 

I’ll submit a patch, and maybe later, I can add the other distribution strategy option.

Reply:
I opened a PR to fix this, small one-liner:

https://github.com/coinjar/react-native-wagmi-charts/pull/23#issue-1040493727